src/redux/sagas.js

```diff
@@ -15,21 +16,47 @@ import {
   fetchFormsFailure,
 } from './actions';
 
-import gapi, { CLIENT_ID, DISCOVERY_DOCS, SCOPES, SCRIPT_ID } from '../services/google';
+import gapi, { CLIENT_ID_THEODO_FR, CLIENT_ID_THEODO_UK, CLIENT_ID_FASTIT, DISCOVERY_DOCS, SCOPES, SCRIPT_ID_THEODO_FR, SCRIPT_ID_THEODO_UK, SCRIPT_ID_FASTIT } from '../services/google';
+
+var clientId, scriptId
+
+function selectCompany(company) {
```

Comment:
I think this function should return `{ clientId, scriptId }` and then in the sagas you can define: `const { clientId } = selectCompany(action.company)`. In that way we avoid global variables.

Another way would be to store the clientId and the scriptId in the store and update them on the `COMPANY_SELECTED` action. You can then access them in the saga using `select`.